src/transformers/models/layoutlmv2/re.py

```diff
@@ -0,0 +1,154 @@
+import copy
+
+import torch
+from torch import nn
+from torch.nn import CrossEntropyLoss
+
+
+class BiaffineAttention(torch.nn.Module):
```

Comment:
```suggestion
class LayoutLMv2BiaffineAttention(torch.nn.Module):
```
We usually add a model-specific prefix to each class.

Reply:
@NielsRogge 
`self.rel_classifier = BiaffineAttention(config.hidden_size // 2, 2)`
Can you explain why we pass those parameters to BiaffineAttention class?
2 because we it should classify as [(QUESTION, ANSWER), None]?

Reply:
Let's say that we have 5 labels [other, L1, L2, L3, L4] and we don't have a predefined relation for them like [Question, Answer] in this case. Does it mean that I need to pass a number of all the possible combinations (4 * 3 = 12) instead of 2?